firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/internal/ModelFileDownloadService.java

```diff
@@ -253,8 +299,8 @@ private synchronized ParcelFileDescriptor getDownloadedFile(Long downloadingId)
     try {
       fileDescriptor = downloadManager.openDownloadedFile(downloadingId);
     } catch (FileNotFoundException e) {
-      // todo(annz)
-      System.out.println("Downloaded file is not found");
+      // todo replace with FirebaseMlException
+      Log.d(TAG, "Downloaded file is not found" + e);
```

Comment:
Dot or space after "found" so it doesn't get mashed together with the exception message

Reply:
done